src/components/common/ThreeToggle/index.tsx

```diff
@@ -0,0 +1,127 @@
+import { useTheme } from 'next-themes';
+import { useEffect, useId, useState } from 'react';
+import MonitorIcon from '../../../../public/assets/icons/monitor.svg';
+import MoonIcon from '../../../../public/assets/icons/moon.svg';
+import SunIcon from '../../../../public/assets/icons/sun.svg';
+import styles from './style.module.scss';
+
+const ThreeToggle = () => {
+  const { theme, setTheme } = useTheme();
+  const [mounted, setMounted] = useState(false);
+
+  const lightId = `light${useId()}`;
+  const systemId = `system${useId()}`;
+  const darkId = `light${useId()}`;
```

Comment:
```suggestion
  const darkId = `dark${useId()}`;
```
should this say dark?